src/views/ideas/l10n-static.json

```diff
@@ -0,0 +1,15 @@
+{
+    "cards.ScratchCardsAllLink": "https://resources.scratch.mit.edu/www/cards/en/scratch-cards-all.pdf",
+    "cards.nameCardsLink": "https://resources.scratch.mit.edu/www/cards/en/name-cards.pdf",
+    "cards.animateCardsLink": "https://resources.scratch.mit.edu/www/cards/en/animate-cards.pdf",
+    "cards.musicCardsLink": "https://resources.scratch.mit.edu/www/cards/en/music-cards.pdf",
+    "cards.storyCardsLink": "https://resources.scratch.mit.edu/www/cards/en/story-cards.pdf",
+    "cards.chaseCardsLink": "https://resources.scratch.mit.edu/www/cards/en/chase-cards.pdf",
+    "cards.videoCardsLink": "https://resources.scratch.mit.edu/www/cards/en/video-cards.pdf",
+    "guides.nameGuideLink": "https://resources.scratch.mit.edu/www/guides/en/NameGuide.pdf",
```

Comment:
So, unfortunately the cards and the guides did not use the same naming conventions on the resources server, so the script that generates the names (based on filename) uses uppercase for guides: `guides.NameGuideLink`. We should probably fix this. I'm going to have to revise the script that generates the data for the cards in any case. I'll look into whether I can standardize the Guides link.

Reply:
On second thoughts, let's make them work with the script as is. So the ones that exist already need to be capitalized:
- `guides.NameGuideLink`
- `guides.MusicGuideLink`
- `guides.StoryGuideLink`

The script that generates the list simply takes the filename without extension, and appends `Link`, so we need to make sure we coordinate with Natalie on naming the ones that are new.